clients/go/test/shell/Dockerfile

```diff
@@ -25,6 +25,8 @@ COPY --from=builder /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
 COPY --from=builder /go/bin/app /app
 # Copy the client config.
 COPY clients/go/test/shell/config.yaml /etc/lumberjack/config.yaml
+COPY integration/testrunner/public_key.json public_key.json
```

Comment:
Not sure whether it is a good practise to store the public key in the root directory of the docker container?

Reply:
Chen and I discussed it yesterday about this, and decide to use this.